include/deal.II/fe/fe_dgp_nonparametric.h

```diff
@@ -289,63 +289,81 @@ class FE_DGPNonparametric : public FiniteElement<dim,spacedim>
   requires_update_flags (const UpdateFlags update_flags) const;
 
   /**
-   * Return the value of the @p ith shape function at the point @p p. See the
-   * FiniteElement base class for more information about the semantics of this
-   * function.
+   * This function is intended to return the value of a shape function
+   * at a point on the reference cell. However, since the current element
+   * does not implement shape functions by mapping from a reference cell,
+   * no shape functions exist on the reference cell.
+   *
+   * Consequently, as discussed in the corresponding function in the base
+   * class, FiniteElement::shape_value(), this function throws an exception
+   * of type FiniteElement::ExcUnitShapeValuesDoNotExist.
    */
   virtual double shape_value (const unsigned int i,
                               const Point<dim> &p) const;
 
   /**
-   * Return the value of the @p componentth vector component of the @p ith
-   * shape function at the point @p p. See the FiniteElement base class for
-   * more information about the semantics of this function.
+   * This function is intended to return the value of a shape function
```

Comment:
The function always throws an exception, so I didn't want to go into details of what exactly the arguments mean. It's also documented in the base class to which I refer here.